spec/models/drawing_spec.rb

```diff
@@ -27,18 +27,6 @@
         .is_less_than_or_equal_to(5)
     end
 
-    context "images/paperclip" do
```

Comment:
As you mentioned, will be nice to replace these once we figure out validation in Carrierwave!